mocap_optitrack_driver_node:
  ros__parameters:
    marker_pub: true
    rigid_pub: true
    connection_type: "Unicast" # Unicast / Multicast
    server_address: "198.17.0.120"
    local_address: "198.17.0.48"
    multicast_address: "239.255.42.99"
    server_command_port: 1510
    server_data_port: 1511
    rigid_body_name: "ground"
    lastFrameNumber: 0
    frameCount: 0
    droppedFrameCount: 0
    n_markers: 0
    n_unlabeled_markers: 0
    qos_history_policy: "keep_all"         # keep_all / keep_last
    qos_reliability_policy: "best_effort"  # best_effort / reliable
    qos_depth: 10                         # 10 / 100 / 1000
    serial: true
    serial_port: "/dev/ttyUSB0"
    serial_buad: 57600

optitrack_serial:
  ros__parameters:
    rigid_id: 1
    topic_name: "/mavros/vision_pose/pose"
    ridids_pub: false
    serial_port: "/dev/ttyUSB0"
    serial_buad: 57600
    